lib/AST/ASTImporterLookupTable.cpp

```diff
@@ -30,13 +30,13 @@ struct Builder : RecursiveASTVisitor<Builder> {
   bool VisitFriendDecl(FriendDecl *D) {
     if (D->getFriendType()) {
       QualType Ty = D->getFriendType()->getType();
-      // FIXME Can this be other than elaborated?
-      QualType NamedTy = cast<ElaboratedType>(Ty)->getNamedType();
-      if (!NamedTy->isDependentType()) {
-        if (const auto *RTy = dyn_cast<RecordType>(NamedTy))
+      if (isa<ElaboratedType>(Ty))
```

Comment:
Are here no other cases possible?

Reply:
I don't know. There may be. So I am going to add an "llvm_unreachable" before commit.